### PR TITLE
section 397, 401, 407, 408, 410, 412, 421, 429

### PR DESCRIPTION
**Pull request by lucasmillanm**
### What?

section 397, 401, 407, 408, 410, 412, 421, 429

### Why?

simple sections, that had to be done

### How?

yaml implementation

### Testing?

no

### Anything Else?

[//]: # (for a sample see: https://www.pullrequest.com/blog/writing-a-great-pull-request-description/)

### CODE REVIEW

File: fabbwled-backend/src/main/resources/sections/397-432.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      - turnTo: 10
+    - text: Head for Trefoille
+      then:
+      - turnTo: 250
+
+
+- number: 421
+  content:
+  - text: The storm demons shrink back with a wailing, mournful cry, like wind in the trees. The power of your faith is enough to repel them. You work free one of the stakes holding down Sul Veneris. Turn to 365.
+  - turnTo: 250
```
Comment (Desi7) [inline fabbwled-backend/src/main/resources/sections/397-432.yaml:48]: shouldn't it be the same page?

**Review by Desi7 — CHANGES_REQUESTED** [review]
One small change, rest looks good to me 👍

**Review by Rwael — APPROVED** [review]
Looks good 😊 I think future branch name prefixes should be section/ or just name like lucas/?

File: fabbwled-backend/src/main/resources/sections/397-432.yaml (hunk 1 above)
Comment (lucasmillanm) [reply fabbwled-backend/src/main/resources/sections/397-432.yaml]: little mistake.

**Review by Desi7 — APPROVED** [review]
thx 👍

File: fabbwled-backend/src/main/resources/sections/397-432.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+- number: 397
+  content:
+    - text: Your ship is thrown about like flotsam and jetsam. When the storm subsides, you take stock. Much has been swept overboard – you lose 1 Cargo Unit, if you had any, of your choice. Also, the ship has been swept way off course and the mate has no idea where you are. ‘We’re lost at sea, Cap’n!’ he moans. Turn to 90.
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/397-432.yaml:3]: Can you remove the `Turn to 90` everywhere?

**Review by Nilstrieb — CHANGES_REQUESTED** [review]

File: fabbwled-backend/src/main/resources/sections/397-432.yaml (hunk 2 above)
Comment (lucasmillanm) [reply fabbwled-backend/src/main/resources/sections/397-432.yaml]: yessir, done

**Review by Nilstrieb — APPROVED** [review]
looks great!

**Review by Lion-alt — APPROVED** [review]
Looks good

**Review by adrsch04 — APPROVED** [review]
looks good

**Pull request merged**